modules/ROOT/pages/configuring-properties.adoc

```diff
@@ -229,8 +229,9 @@ Configuration properties can be overwritten. The Hierarchy in which these are tr
 . System Properties
 . Environment Properties
 . Deployment Properties
-. Application Properties
+. Application Properties (includes Configuration Properties, Secure Configuration Properties and other Custom Configuration Properties)
+. Global Properties
 
-So, for example, if a configuration property is defined in a System Property, and there is also an application configuration property, the value for that application is the last one.
```

Comment:
So, for example if a configuration property is defined as a system property, and there is also an application configuration of the same name, the application uses the value of the system property, since that is evaluated before the application property.